src/app/components/cds/forms/LanguagePickerSelect.js

```diff
@@ -97,6 +100,7 @@ LanguagePickerSelect.propTypes = {
   languages: PropTypes.arrayOf(PropTypes.string),
   isDisabled: PropTypes.bool,
   showLabel: PropTypes.bool,
+  helpContent: PropTypes.element,
```

Comment:
Added helpcontent to this component to use in some upcoming help text changes from Noah